org-formation/800-redirects/_tasks.yaml

```diff
@@ -19,3 +19,20 @@ SsoRedirect:   # Redirect sso.sageit.org to the AWS SSO Start URL
     TargetHostName: "d-906769aa66.awsapps.com"
     # and a path to our specific config
     TargetKey: "start#/"
+
+FinopsApiRedirect:  # redirect finops-api.sageit.org to lambda-mips-api cloudfront distribution
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.6.3/templates/R53/cname.yaml
```

Comment:
The s3-redirector template is not appropriate in this case because we're not hosting static files out of an S3 bucket and we have already created the Cloudfront distribution in the lambda template. In the lambda use-case, we only need to create a CNAME record in Route 53 for the Cloudfront distribution we've already created in another stack, so I chose to create a new template for that use case.

Reply:
ok